### DISCUSSION TITLE
[Projects Beta] Scrolling in table view is glitchy

### DISCUSSION THREAD

**Discussion by minhduc0711** [community/community#6510] [Projects and Issues] [answered]
It feels like the list is always in a loading state whenever I scroll the page, and sometimes the glitches make it unusable (the later part of the video).

https://user-images.githubusercontent.com/34294427/137836543-e3e1610f-b608-4ca8-8b78-912ac47c2a63.mp4

**Comment by azenMatt** [comment, marked as answer]
We just got a fix into a PR that should solve this - it should be with you shortly 👍
